core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -618,7 +618,15 @@ angular.module('oppia').directive('questionsList', [
                       });
                     });
                   };
-                  $scope.saveAndcommit = function() {
+                  // the saveAndCommit function is called when the contents of
```

Comment:
Good comment, but use proper punctuation. Start with a capital letter.

Also drop the newline at the end -- comments should be adjacent to the functions they describe. You can also add a newline before the start of the comment to separate saveAndCommit() from the previous function.

Reply:
done